sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadClient.java

```diff
@@ -84,21 +102,21 @@
 
     /**
      * Sends a message to a thread.
-     *
+     * 
      * @param chatThreadId The thread id to send the message to.
      * @param sendChatMessageRequest Details of the message to send.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws ErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the ID of the message sent.
+     * @return result of the send message operation.
      */
-    public Response<String> sendChatMessageWithRestResponse(String chatThreadId, SendChatMessageRequest sendChatMessageRequest) {
+    public Response<SendChatMessageResult> sendChatMessageWithRestResponse(String chatThreadId, SendChatMessageRequest sendChatMessageRequest) {
```

Comment:
We should add simple convert here to change Response<SendChatMessageResult> to Response<String> which is the message id. Remember also update JavaDoc

Reply:
Made manual change to convenience layer.